spec/index.html

```diff
@@ -3775,6 +3775,107 @@ <h2>Acknowledgements</h2>
+      of a graph, J. Algorithms 5 (1984) 345-362.
+    </li>
+    <li>
+      G. Sutcliffe, C. Suttner, The TPTP problem library: CNF release v1.2.1, J.
+      Automat. Reason. 21 (2) (1998) 177-203.
+    </li>
+    <li>
+      M. Grohe, Isomorphism testing for embeddable graphs through definability,
+      in: STOC ’00: Proceedings of the Thirty-Second Annual ACM Symposium on
+      Theory of Computing, ACM, New York, NY, USA, 2000, pp. 63-72,
+      http://doi.acm.org/10.1145/335305.335313.
+    </li>
+    <li>
+      W.W. Tsang, Tuning the collision test for stringency, Tech. Rep.
+      TR-2000-05, Computer Science and Information Systems, Hong Kong
+      University, 2000.
+    </li>
+    <li>
+      Tim Berners-Lee and Dan Connolly, [[[DesignIssues-Diff]]]
+      [[DesignIssues-Diff]] (2001)
+    </li>
+    <li>
+      Jeremy J. Carroll, [[[HPL-2003-142]]] [[HPL-2003-142]] (2003)
+    </li>
+    <li>
+      J.J. Carroll, Signing RDF graphs, in: The Semantic Web - ISWC 2003, Second
+      International Semantic Web Conference, in: Lecture Notes in Comput.
+      Sci.,vol. 2870, Springer, 2003, pp. 369-384.
+    </li>
+    <li>
+      J.L. Gross, J. Yellen (Eds.), Handbook of Graph Theory, Discrete
+      Mathematics and Its Applications, CRC Press, 2003.
+    </li>
+    <li>
+      C. Sayers, A. Karp, Computing the digest of an RDF graph, Tech. Rep.
+      HPL-2003-235 (R. 1), Hewlett Packard Laboratories, 2004
+    </li>
+    <li>
+      G. Tummarello, C. Morbidoni, R. Bachmann-Gmür, O. Erling, RDFSync:
+      efficient remote synchronization of RDF models, in: The Semantic Web, 6th
+      International Semantic Web Conference, Busan, Korea, November 11-15, 2007,
+      in: Lecture Notes in Comput. Sci., vol. 4825, Springer, 2007.
+    </li>
+    <li>
+      E. Oren, R. Delbru, M. Catasta, R. Cyganiak, H. Stenzhorn, G. Tummarello,
+      Sindice.com: a document-oriented lookup index for open linked data,
+      International Journal of Metadata, Semantics and Ontologies 3 (1) (2008)
+      37-52.
+    </li>
+    <li>
+      Jesus Arias Fisteus, Norberto Fernández García, Luis Sánchez Fernández,
+      Carlos Delgado Kloos, Hashing and canonicalizing Notation 3 graphs,
+      Journal of Computer and System Sciences, 2010 Elsevier Inc Depto. de
+      Ingeniería Telemática, Universid
+    </li>
+    <li>
+      Kasten, A., Scherp, A., Schauß, P. (2014). [[[eswc2014Kasten]]]. In:
+      Presutti, V., d’Amato, C., Gandon, F., d’Aquin, M., Staab, S., Tordai, A.
+      (eds) The Semantic Web: Trends and Challenges. ESWC 2014. Lecture Notes in
+      Computer Science, vol 8465. Springer, Cham.
+      https://doi.org/10.1007/978-3-319-07443-6_11 [[eswc2014Kasten]]
+    </li>
+    <li>
+      Aidan Hogan. 2017. Canonical Forms for Isomorphic and Equivalent RDF
+      Graphs: Algorithms for Leaning and Labelling Blank Nodes. ACM Trans. Web
+      0, 0, Article 00 (January 2017), 61 pages. [[Hogan-Canonical-RDF]]
+    </li>
+    <li>
```

Comment:
Not sure if the formatting is correct, reference taken [from here](https://www.w3.org/TR/rch-explainer/#informative-references) and rewritten to try and match the style used here. It was respec-ified in the other place.

```suggestion
    </li>
    <li>
      Rachel Arnold, Dave Longley. 2020. RDF Dataset Normalization. Report
      submitted to the W3C Credentials Community Group mailing list.
      https://lists.w3.org/Archives/Public/public-credentials/2021Mar/att-0220/RDFDatasetCanonicalization-2020-10-09.pdf [[arnold-longley-2020]]
    </li>
    <li>
```

Reply:
This is effectively the same as the `[CCG-RDC-FINAL]` reference. I'm fine for spelling it out, but be may want to make the two different references consistent.

The source for CCG-RDC-FINAL is in common/biblio.js.